pkgs/top-level/all-packages.nix

```diff
@@ -32241,10 +32241,11 @@ with pkgs;
       })
     nix
     nixStable
+    nix_2_3
     nix_2_4
     nixUnstable;
 
-  nixStatic = pkgsStatic.nix;
+  nixStatic = pkgsStatic.nix_2_3;
```

Comment:
Error log:

<details>
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/kp5jz4aqx7pp0mjg34153md8mz2k116h-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "autoreconfPhase" }
autoreconfPhase
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: /nix/store/ab270ciikwdj695g24rv7x86kkwb6sdb-autoconf-2.71/bin/autoconf --force
configure.ac:1: warning: AC_INIT: not a literal: "m4_esyscmd(bash -c "echo -n $(cat ./.version)$VERSION_SUFFIX")"
autoreconf: running: /nix/store/ab270ciikwdj695g24rv7x86kkwb6sdb-autoconf-2.71/bin/autoheader --force
autoreconf: configure.ac: not using Automake
autoreconf: Leaving directory '.'
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
Updating Autotools / GNU config script to a newer upstream version: ./config/config.sub
Updating Autotools / GNU config script to a newer upstream version: ./config/config.guess
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
configure flags: --prefix=/nix/store/cpsd1l2p7yzj5r60jyyz7fxyq3p4yy0h-nix-static-x86_64-unknown-linux-musl-2.4 --bindir=/nix/store/cpsd1>
configure: WARNING: unrecognized options: --enable-static
checking for a sed that does not truncate output... /nix/store/d9drqi4daha3f0b6wm5y0fnabbggy1r2-gnused-4.8/bin/sed
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-musl
checking for the canonical Nix system name... x86_64-linux
checking for x86_64-unknown-linux-musl-gcc... x86_64-unknown-linux-musl-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables...
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether x86_64-unknown-linux-musl-gcc accepts -g... yes
checking for x86_64-unknown-linux-musl-gcc option to enable C11 features... none needed
checking whether the compiler supports GNU C++... yes
checking whether x86_64-unknown-linux-musl-g++ accepts -g... yes
checking for x86_64-unknown-linux-musl-g++ option to enable C++11 features... none needed
checking how to run the C preprocessor... x86_64-unknown-linux-musl-gcc -E
checking for x86_64-unknown-linux-musl-ar... x86_64-unknown-linux-musl-ar
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking for struct dirent.d_type... yes
checking for pubsetbuf... yes
checking for statvfs... yes
checking for pipe2... yes
checking for lutimes... yes
checking whether it is possible to create a link to a symlink... yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for locale... yes
checking for bash... /nix/store/phqa311klldrcbwid1i22dwnpfc9dnma-bash-5.1-p8/bin/bash
checking for flex... /nix/store/k64ipsghczdyf0pirbvqmjk4idkh6860-flex-2.6.4/bin/flex
checking for bison... /nix/store/5nhdzkfiv2cin7hlhq88i8zjhgbi82hn-bison-3.7.6/bin/bison
checking for dot... no
checking for lsof... lsof
checking for jq... /nix/store/22r0xq2brgk0xha0djg0s3sznhrpa5hf-jq-1.6-bin/bin/jq
checking for boostlib >= 1.66 (106600)... yes
checking whether -latomic is needed... no
checking pkg-config is at least version 0.9.0... yes
checking for libcrypto... yes
checking for libarchive >= 3.1.2... yes
checking for sqlite3 >= 3.6.19... yes
checking for libcurl... yes
checking for libeditline... yes
checking for libsodium... yes
checking for libbrotlienc libbrotlidec... yes
checking for libcpuid... yes
checking for libseccomp... yes
checking for aws/s3/S3Client.h... no
checking for bdw-gc... yes
checking for gtest_main... yes
checking for lowdown >= 0.8.0... yes
checking for setresuid... yes
checking for setreuid... yes
checking for lchown... yes
checking for strsignal... yes
checking for posix_fallocate... yes
checking for sysconf... yes
configure: creating ./config.status
config.status: creating config.h
configure: WARNING: unrecognized options: --enable-static
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j6 -l6 SHELL=/nix/store/phqa311klldrcbwid1i22dwnpfc9dnma-bash-5.1-p8/bin/bash profiledir=\$\(out\)/etc/profile.d PRECOMPIL>
  GEN    Makefile.config
  CXX    src/libutil/tests/closure.o
  CXX    src/libutil/tests/compression.o
  CXX    src/libutil/tests/config.o
  CXX    src/libutil/tests/hash.o
  CXX    src/libutil/tests/json.o
  CXX    src/libutil/tests/logging.o
  CXX    src/libutil/tests/lru-cache.o
  CXX    src/libutil/tests/pool.o
  CXX    src/libutil/tests/tests.o
  CXX    src/libutil/tests/url.o
  CXX    src/libutil/tests/xml-writer.o
  CXX    src/libutil/affinity.o
  CXX    src/libutil/archive.o
  CXX    src/libutil/args.o
  CXX    src/libutil/compression.o
  CXX    src/libutil/compute-levels.o
  CXX    src/libutil/config.o
  CXX    src/libutil/error.o
  CXX    src/libutil/hash.o
  CXX    src/libutil/json.o
  CXX    src/libutil/logging.o
  CXX    src/libutil/rust-ffi.o
  CXX    src/libutil/serialise.o
  CXX    src/libutil/tarfile.o
  CXX    src/libutil/thread-pool.o
  CXX    src/libutil/url.o
  CXX    src/libutil/util.o
  CXX    src/libutil/xml-writer.o
  CXX    src/libstore/tests/references.o
  CXX    src/libstore/binary-cache-store.o
  CXX    src/libstore/build/derivation-goal.o
  CXX    src/libstore/build/drv-output-substitution-goal.o
  CXX    src/libstore/build/entry-points.o
  CXX    src/libstore/build/goal.o
  CXX    src/libstore/build/hook-instance.o
  CXX    src/libstore/build/local-derivation-goal.o
  CXX    src/libstore/build/substitution-goal.o
  CXX    src/libstore/build/worker.o
  CXX    src/libstore/builtins/buildenv.o
  CXX    src/libstore/builtins/fetchurl.o
  CXX    src/libstore/builtins/unpack-channel.o
  CXX    src/libstore/content-address.o
  CXX    src/libstore/crypto.o
  CXX    src/libstore/daemon.o
  CXX    src/libstore/derivations.o
  CXX    src/libstore/derived-path.o
  CXX    src/libstore/dummy-store.o
  CXX    src/libstore/export-import.o
  CXX    src/libstore/filetransfer.o
  CXX    src/libstore/gc.o
  CXX    src/libstore/globals.o
  CXX    src/libstore/http-binary-cache-store.o
  CXX    src/libstore/legacy-ssh-store.o
  CXX    src/libstore/local-binary-cache-store.o
  CXX    src/libstore/local-fs-store.o
  GEN    src/libstore/schema.sql.gen.hh
  GEN    src/libstore/ca-specific-schema.sql.gen.hh
  CXX    src/libstore/lock.o
  CXX    src/libstore/machines.o
  CXX    src/libstore/misc.o
  CXX    src/libstore/names.o
  CXX    src/libstore/nar-accessor.o
  CXX    src/libstore/nar-info-disk-cache.o
  CXX    src/libstore/nar-info.o
  CXX    src/libstore/optimise-store.o
  CXX    src/libstore/parsed-derivations.o
  CXX    src/libstore/path-info.o
  CXX    src/libstore/path-with-outputs.o
  CXX    src/libstore/path.o
  CXX    src/libstore/pathlocks.o
  CXX    src/libstore/profiles.o
  CXX    src/libstore/realisation.o
  CXX    src/libstore/references.o
  CXX    src/libstore/remote-fs-accessor.o
  CXX    src/libstore/remote-store.o
  CXX    src/libstore/s3-binary-cache-store.o
  CXX    src/libstore/sqlite.o
  CXX    src/libstore/ssh-store.o
  CXX    src/libstore/ssh.o
  CXX    src/libstore/store-api.o
  CXX    src/libstore/uds-remote-store.o
  GEN    src/libexpr/parser-tab.cc
  GEN    src/libexpr/parser-tab.hh
  GEN    src/libexpr/lexer-tab.cc
src/libexpr/lexer.l:295: warning, -s option given but default rule can be matched
  GEN    src/libexpr/lexer-tab.hh
src/libexpr/lexer.l:295: warning, -s option given but default rule can be matched
  GEN    src/nix-channel/unpack-channel.nix.gen.hh
  CXX    src/nix-collect-garbage/nix-collect-garbage.o
  CXX    src/nix-copy-closure/nix-copy-closure.o
  CXX    src/nix-env/nix-env.o
  GEN    src/nix-env/buildenv.nix.gen.hh
  CXX    src/nix-instantiate/nix-instantiate.o
  CXX    src/nix-store/dotgraph.o
  CXX    src/nix-store/graphml.o
  CXX    src/nix-store/nix-store.o
  CXX    src/nix/add-to-store.o
  CXX    src/nix/app.o
  CXX    src/nix/build.o
  CXX    src/nix/bundle.o
  CXX    src/nix/cat.o
  CXX    src/nix/copy.o
  CXX    src/nix/daemon.o
  CXX    src/nix/describe-stores.o
  GEN    src/nix/get-env.sh.gen.hh
  CXX    src/nix/diff-closures.o
  CXX    src/nix/doctor.o
  CXX    src/nix/dump-path.o
  CXX    src/nix/edit.o
  CXX    src/nix/eval.o
  CXX    src/nix/flake.o
  CXX    src/nix/hash.o
  CXX    src/nix/log.o
  CXX    src/nix/ls.o
  GEN    doc/manual/generate-manpage.nix.gen.hh
  GEN    doc/manual/utils.nix.gen.hh
  CXX    src/nix/make-content-addressable.o
  CXX    src/nix/nar.o
  CXX    src/nix/optimise-store.o
  CXX    src/nix/path-info.o
  CXX    src/nix/ping-store.o
  CXX    src/nix/prefetch.o
  CXX    src/nix/profile.o
  CXX    src/nix/realisation.o
  CXX    src/nix/registry.o
  CXX    src/nix/repl.o
  CXX    src/nix/run.o
  CXX    src/nix/search.o
  CXX    src/nix/show-config.o
  CXX    src/nix/show-derivation.o
  CXX    src/nix/sigs.o
  CXX    src/nix/store-delete.o
  CXX    src/nix/store-gc.o
  CXX    src/nix/store-repair.o
  CXX    src/nix/store.o
  CXX    src/nix/upgrade-nix.o
  CXX    src/nix/verify.o
  CXX    src/libexpr/common-eval-args.o
  CXX    src/libexpr/eval-cache.o
  CXX    src/libexpr/eval.o
  CXX    src/libexpr/attr-set.o
  CXX    src/libexpr/attr-path.o
  CXX    src/nix/why-depends.o
  CXX    src/libexpr/flake/config.o
  GEN    src/libexpr/flake/call-flake.nix.gen.hh
  CXX    src/libexpr/flake/flakeref.o
  CXX    src/libexpr/json-to-value.o
  CXX    src/libexpr/get-drvs.o
  CXX    src/libexpr/lexer-tab.o
  CXX    src/libexpr/function-trace.o
  CXX    src/libexpr/flake/lockfile.o
  CXX    src/libexpr/nixexpr.o
  CXX    src/libexpr/parser-tab.o
  GEN    src/libexpr/imported-drv-to-derivation.nix.gen.hh
  GEN    src/libexpr/primops/derivation.nix.gen.hh
  GEN    src/libexpr/fetchurl.nix.gen.hh
  CXX    src/libexpr/primops/context.o
  CXX    src/libexpr/primops/fetchMercurial.o
  CXX    src/libexpr/value-to-json.o
  CXX    src/libexpr/value-to-xml.o
  CXX    src/libmain/common-args.o
  CXX    src/libexpr/primops/fromTOML.o
  CXX    src/libexpr/primops/fetchTree.o
  CXX    src/libmain/loggers.o
  CXX    src/libmain/progress-bar.o
  CXX    src/libmain/shared.o
  CXX    src/libmain/stack.o
  CXX    src/libfetchers/attrs.o
  CXX    src/libfetchers/cache.o
  CXX    src/libfetchers/fetchers.o
  CXX    src/libfetchers/git.o
  CXX    src/libfetchers/github.o
  CXX    src/libfetchers/indirect.o
  CXX    src/libfetchers/mercurial.o
  CXX    src/libfetchers/path.o
  CXX    src/libfetchers/registry.o
  CXX    src/libfetchers/tarball.o
  CXX    src/libcmd/command.o
  CXX    src/libcmd/installables.o
  CXX    src/libcmd/legacy.o
  CXX    src/libcmd/markdown.o
  LD     src/libutil/libnixutil.a
  CXX    src/libstore/local-store.o
  GEN    scripts/nix-reduce-build
  GEN    scripts/nix-profile.sh
  CXX    src/build-remote/build-remote.o
  AR     src/libutil/libnixutil.a
  CXX    src/nix-build/nix-build.o
  GEN    scripts/nix-http-export.cgi
  CXX    src/nix-channel/nix-channel.o
  CXX    src/nix-env/user-env.o
  CXX    src/nix/develop.o
  CXX    src/nix/main.o
  CXX    src/libexpr/flake/flake.o
  CXX    src/libexpr/primops.o
  LD     src/libmain/libnixmain.a
  AR     src/libmain/libnixmain.a
  LD     src/libfetchers/libnixfetchers.a
  AR     src/libfetchers/libnixfetchers.a
  LD     src/libcmd/libnixcmd.a
  AR     src/libcmd/libnixcmd.a
  LD     src/libutil/tests/libutil-tests
  LD     src/libstore/libnixstore.a
  AR     src/libstore/libnixstore.a
  LD     src/libstore/tests/libstore-tests
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:352: multiple definition of `OPENSSL_instrument_bus2'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwh>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/ctype.c:267: multiple definition of `ossl_tolower'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-sta>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha256.c:79: multiple definition of `SHA224_Update'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openss>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/ctype.c:272: multiple definition of `ossl_toupper'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-sta>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha256.c:50: multiple definition of `SHA224'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-stati>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:323: multiple definition of `SHA512_Transform'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-op>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:188: multiple definition of `SHA1_Transform'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhl>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha1-x86_64.s:9: multiple definition of `sha1_block_data_order'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwh>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/cryptlib.c:400: multiple definition of `OPENSSL_showfatal'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-ope>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha_local.h:55: multiple definition of `SHA1_Init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:135: multiple definition of `SHA256_Update'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlq>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:141: multiple definition of `SHA512_Final'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openss>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/md5/md5-x86_64.s:8: multiple definition of `md5_block_asm_data_order'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpb>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/cryptlib.c:413: multiple definition of `OPENSSL_isservice'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-ope>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha256.c:21: multiple definition of `SHA224_Init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl->
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:44: multiple definition of `OPENSSL_ia32_cpuid'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4->
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/cryptlib.c:417: multiple definition of `OPENSSL_die'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-s>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:192: multiple definition of `SHA256_Final'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqd>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:261: multiple definition of `SHA384_Final'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openss>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:135: multiple definition of `SHA1_Update'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:192: multiple definition of `SHA1_Final'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:188: multiple definition of `SHA256_Transform'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbw>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:101: multiple definition of `SHA384_Init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:32: multiple definition of `OPENSSL_rdtsc'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-opens>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:341: multiple definition of `SHA512'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-stat>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:314: multiple definition of `SHA384_Update'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-opens>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:119: multiple definition of `SHA512_Init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:401: multiple definition of `OPENSSL_ia32_rdrand_bytes'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpb>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/md5/md5_dgst.c:25: multiple definition of `MD5_Init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-s>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:188: multiple definition of `MD5_Transform'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlq>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:285: multiple definition of `OPENSSL_wipe_cpu'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-o>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:192: multiple definition of `MD5_Final'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4->
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:270: multiple definition of `SHA512_Update'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-opens>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha256.c:84: multiple definition of `SHA224_Final'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512-x86_64.s:9: multiple definition of `sha512_block_data_order'; /nix/store/cdxhdnzh0dw2mxjvhlka82l>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha256-x86_64.s:9: multiple definition of `sha256_block_data_order'; /nix/store/cdxhdnzh0dw2mxjvhlka82l>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:65: multiple definition of `sha512_224_init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-open>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/ctype.c:277: multiple definition of `ascii_isdigit'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-st>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:16: multiple definition of `OPENSSL_atomic_add'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4->
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/cryptlib.c:104: multiple definition of `OPENSSL_cpuid_setup'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-o>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:213: multiple definition of `OPENSSL_cleanse'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-op>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha256.c:36: multiple definition of `SHA256_Init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl->
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/ctype.c:257: multiple definition of `ossl_ctype_check'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:327: multiple definition of `SHA384'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-stat>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:444: multiple definition of `OPENSSL_ia32_rdseed_bytes'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpb>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha512.c:83: multiple definition of `sha512_256_init'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-open>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/sha/sha256.c:64: multiple definition of `SHA256'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-openssl-stati>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:318: multiple definition of `OPENSSL_instrument_bus'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhl>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/crypto/x86_64cpuid.s:250: multiple definition of `CRYPTO_memcmp'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4-open>
/nix/store/wir2a9c37zd0xnn5df2chjvpq7s859v8-x86_64-unknown-linux-musl-binutils-2.35.1/bin/x86_64-unknown-linux-musl-ld: src/libutil/libn>
/build/openssl-1.1.1l/include/crypto/md32_common.h:135: multiple definition of `MD5_Update'; /nix/store/cdxhdnzh0dw2mxjvhlka82lpbwhlqdb4>
collect2: error: ld returned 1 exit status
make: *** [mk/lib.mk:118: src/libstore/tests/libstore-tests] Error 1
make: *** Waiting for unfinished jobs....
</details>

Reply:
I think I fixed it, let's see.